File: report-generator/src/main/java/com/teamscale/report/EDuplicateClassFileBehavior.java
package com.teamscale.report;

/**
 * Behavior when two non-identical class files with the same package name are found.
 */
public enum EDuplicateClassFileBehavior {
	/** Completely ignores it. */
	IGNORE,

	/** Prints a warning to the logger. */
	WARN,

	/** Fails and stops further processing. */
	FAIL
}

File: report-generator/src/main/kotlin/com/teamscale/report/EDuplicateClassFileBehavior.kt
package com.teamscale.report

/**
 * Behavior when two non-identical class files with the same package name are found.
 */
enum class EDuplicateClassFileBehavior {
	/** Completely ignores it.  */
	IGNORE,

	/** Prints a warning to the logger.  */
	WARN,

	/** Fails and stops further processing.  */
	FAIL
}
